closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -193,7 +193,7 @@ protected int engineGetOutputSize(int inputLen) {
      */
     @Override
     protected byte[] engineGetIV() {
-        return (nonce == null)? null : nonce.clone();
+        return (initialized && nonce != null) ? nonce.clone() : null;
```

Comment:
If `initialized` is `false`, a `null` will be returned even `nonce` is already set to a non-null value which is not this API intent.

Reply:
I will remove the `initialized` portion of the condition here.